### ISSUE TITLE
Uncaught TypeError: Cannot read property 'openChat' of undefined

### ISSUE THREAD

**Issue by matteoredaelli** [scalableminds/chatroom#118]
Hello

Opening an html page as uìsuggested by your README
I get

  Uncaught TypeError: Cannot read property 'openChat' of undefined

related to html row

  "chatroom.openChat();"

In any case, the chat works properly when I click on the back message title

**Comment by hotzenklotz** [comment]
@matteoredaelli Thanks for the issue. I just tried the `openChat()` method again and had no issues. Please double-check that you initiated the Chatroom into some kind of variable. `openChat()` can only be called on an instance of the `Chatroom` class. See example below:

```
var chatroom = new window.Chatroom({
  title: "Chat with a bot",
  container: document.querySelector(".chat-container"),
  welcomeMessage: "Nice to meet you.",
  host: "http://localhost:5005",
  speechRecognition: "en-US",
  voiceLang: "en-US"
});
chatroom.openChat();
</script>```

@matteoredaelli  Do you have a link to your code/bot?

**Comment by matteoredaelli** [comment]
Now it works using your suggested code instead of the one included in README.md

The difference is "new" 
 var chatroom = new window.Chatroom({

**Comment by jstriebel** [comment]
Thanks for the pointer, we fixed it in #119
